Data/Aeson/Types/FromJSON.hs

```diff
@@ -1140,14 +1151,15 @@ parseNonAllNullarySum p@(tname :* opts :* _) =
     case sumEncoding opts of
       TaggedObject{..} ->
           withObject tname $ \obj -> do
-              tag <- contextType tname $ obj .: tagKey
+              tag <- contextType tname . contextTag tagKey cnames_ $ obj .: tagKey
               fromMaybe (badTag tag <?> Key tagKey) $
-                  parseFromTaggedObject (tag :* contentsFieldName :* p) obj
+                parseFromTaggedObject (tag :* contentsFieldName :* p) obj
```

Comment:
This edit can be avoided.

Reply:
Curious, what are the indenting rules around this. I was using the default Haskell mode indentation in Emacs.

Reply:
We use 4 space indentation (See CONTRIBUTING.md), but we are definitely not consistent about it.

Reply:
Thanks @bergmark. I just noticed that in the CONTRIBUTING.md.  I will rollback my edit.